pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -692,11 +692,21 @@ func sendProtoStrForLogs(reportLogs *logs.LogBundle, image string,
 	}
 	size := int64(proto.Size(reportLogs))
 	if size > logMaxBytes {
-		log.Warnf("sendProtoStrForLogs: %d bytes: %s\n",
-			size, reportLogs)
+		log.Warnf("LogBundle Start: DevID %s, Image %s, EveVersion %s",
```

Comment:
Please add len(reportLogs.Log) to what you log here.

Reply:
will do